be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -412,7 +395,6 @@ Status BetaRowsetWriter::_segcompaction_if_necessary() {
 }
 
 Status BetaRowsetWriter::_segcompaction_ramaining_if_necessary() {
```

Comment:
Since we are actually not compacting anything, I think the function would better to be renamed. Otherwise, it would  be missleading that there is no more waiting after 'segcompaction' (see line 608).

Reply:
Done, I kept segcompaction in name to indicate it's related to segcompaction.